### ISSUE TITLE
What are the versions of tensorfn and pydantic

### ISSUE THREAD

**Issue by wq1315** [ankanbhunia/PIDM#63]

**Comment by ankanbhunia** [comment]
pydantic==1.10.14. 

The demo is now fixed.

**Comment by wq1315** [comment]
That's right, I installed the python3.6 environment according to your requirements. However, pydantic and tensorfn kept reporting errors, and The version of pydantic you requested cannot be installed. But when I used the python3.7 environment, everything was solved. Thank you for your reply
